helm/values-dev.yaml

```
@@ -6,7 +6,15 @@ service:
     metricsMonitor:
       enable: false
     healthCheck:
-      enable: false
+      enable: "true"
+      liveness: 
+        initialDelaySeconds: 15
+        periodSeconds: 5
+      readiness:
+        initialDelaySeconds: 15
+        periodSeconds: 12
+        timeoutSeconds: 10
+        failureThreshold: 10
```

```suggestion
        failureThreshold: 10
        successThreshold: 1
```
Let's be explicit about these settings so there is no confusion as to what each one is configured with. The value I've set for `success` is the default kubernetes would set.